docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Radio.htm

```diff
@@ -73,14 +73,14 @@ <h3>Summary</h3>
 
 	<tr>
 		<td>
-			<pre class="intable prettyprint"><a href="#method-enable">enable</a>() </pre>
+			<pre class="intable prettyprint"><a href="#method-enable">Radio enable</a>() </pre>
 		</td>
 		<td><p>Enables the component.</p></td>
 	</tr>
 
 	<tr>
 		<td>
-			<pre class="intable prettyprint"><a href="#method-value">value</a>() </pre>
+			<pre class="intable prettyprint"><a href="#method-value">{string | number}value</a>() </pre>
```

Comment:
Don't we need a space between "{string | number}" and "value"?

Reply:
Fixed